cmd/kubebuilder/main.go

```diff
@@ -52,7 +52,7 @@ func main() {
 	initproject.AddInit(cmd)
 	version.AddVersion(cmd)
 
-	if util.IsNewVersion() {
+	if util.IsNewVersion() || util.IsProjectNotInitialized() {
 		v1.AddCmds(cmd)
 	} else {
 		v0.AddCmds(cmd)
```

Comment:
Please move func `CheckInstall` to a right place :)

Reply:
Do you mean `IsProjectNotInitialized`?

Reply:
This is the current list of binaries being checked `"kubebuilder-gen", "client-gen", "deepcopy-gen", "gen-apidocs", "informer-gen",
			"openapi-gen", "kubebuilder", "conversion-gen", "defaulter-gen", "lister-gen"`

For v1 commands, the binaries need to be available are `"kubebuilder", "controller-gen"`. Are there any other ones?

Reply:
Talked with @droot, we will keep this check as it is.